simd/unit_tests/include/TestSIMD_MathOps.hpp

```diff
@@ -87,9 +90,16 @@ inline void host_check_all_math_ops(const DataType (&first_args)[n],
   host_check_math_op_all_loaders<Abi>(multiplies(), n, first_args, second_args);
 
   // TODO: Place fallback division implementations for all simd integer types
-  if constexpr (std::is_same_v<DataType, double>)
+  if constexpr (std::is_same_v<DataType, double>) {
     host_check_math_op_all_loaders<Abi>(divides(), n, first_args, second_args);
 
+#if defined(KOKKOS_ARCH_AVX2) || defined(KOKKOS_ARCH_AVX512XEON)
+    host_check_math_op_all_loaders<Abi>(cbrt_op(), n, first_args);
+    host_check_math_op_all_loaders<Abi>(exp_op(), n, first_args);
+    host_check_math_op_all_loaders<Abi>(log_op(), n, first_args);
+#endif
```

Comment:
These are missing for all other scalar types and SIMD types?

Reply:
Yes, these are currently only defined in `AVX2` and `AVX512` `double` simd types:
https://github.com/kokkos/kokkos/blob/04d5c55a433fe735e26d5a4900623152b55fc08b/simd/src/Kokkos_SIMD_AVX2.hpp#L565-L588
https://github.com/kokkos/kokkos/blob/04d5c55a433fe735e26d5a4900623152b55fc08b/simd/src/Kokkos_SIMD_AVX512.hpp#L982-L1005

Looking at arm simd intrinsics, they don't provide these instructions in NEON but are in SVE2, which works with different data types than NEON.